lib/newtabicons.js

```diff
@@ -104,8 +104,8 @@ var addMenuItem = function () {
     menuitem = cm.Item({
       label: 'Copy Thumbnail URL',
       context: [
-        cm.URLContext('about:newtab'),
-        cm.SelectorContext('.newtab-cell')
+        cm.URLContext(['resource://activity-streams/data/content/activity-streams.html#/', 'about:newtab']),
+        cm.SelectorContext('.spotlight-image, .tile-img-container .site-icon-background, .newtab-cell')
```

Comment:
Would it make sense to make this query string a const at the top, instead of passing it around to a bunch of places?
(I think so, particularly if we're going to use the `.newtab-cell` for the selector, like we talked about earlier.)